.gitignore

```diff
@@ -116,6 +116,9 @@ venv.bak/
 # mypy
 .mypy_cache/
 
+# poetry
+poetry.lock
+
```

Comment:
can be removed as this now exists in develop

Reply:
Done